.github/workflows/slsa-builder-go.yml

```diff
@@ -122,7 +167,10 @@ jobs:
         id: cosignid
         env:
           COSIGN_EXPERIMENTAL: "true"
+          # Note: binary name is not verified
```

Comment:
threat here is that someone would place an intermediate action that rewrites binary name?
that way we get "signed" prov over malicious binary?

should be mitigated by checking intermediate steps, or since this is packaged into re-useable workflow it should be ok?